tests/trinity/core/p2p-proto/test_v5_les_protocol_topics.py

```diff
@@ -0,0 +1,12 @@
+from eth.chains.ropsten import ROPSTEN_GENESIS_HEADER
+
+from p2p import discovery
+
+from trinity.protocol.les.proto import LESProtocol, LESProtocolV2
+
+
+def test_get_v5_topic():
```

Comment:
This looks like it could get extracted into a stand alone along with these changes:

https://github.com/ethereum/py-evm/pull/1299/files#diff-068a4555749b83cfb235f8ea0457fab2L373